src/d3a/models/myco_matcher/external_matcher.py

```diff
@@ -0,0 +1,137 @@
+import json
+
+import d3a.constants
+from d3a.d3a_core.redis_connections.redis_area_market_communicator import ResettableCommunicator
+from d3a.models.market import validate_authentic_bid_offer_pair
+from d3a.models.market.market_structures import BidOfferMatch, offer_from_JSON_string, \
+    bid_from_JSON_string
+from d3a.models.myco_matcher.base_matcher import BaseMatcher
+
+
+class ExternalMatcher(BaseMatcher):
+    def __init__(self):
+        super(ExternalMatcher, self).__init__()
```

Comment:
```suggestion
        super().__init__()
```
This is the python 3 version.